README.md

```diff
@@ -0,0 +1,116 @@
+# **ALTERNATE BIT PROTOCOL USING CADMIUM**
+---
+### ORGANINIZATION
+---
+#### Carleton University
+### AUTHORS
+---
+**Originally Developed by**: Cristina Ruiz Martin<br/>
+**Modified by**: Peter Bliznyuk-Kvitko and Shubham Agrawal
+### INTRODUCTION
+---
+ABP (Alternating Bit Protocol) is a communication protocol to ensure reliable transmission through unreliable network. In ABP protocol sender sends a packet and waits for an acknowledgement. If the acknowledgement doesn't arrive within a predefined time, the sender re-sends this packet until it receives an expected acknowledgement and then sends the next packet. In order to distinguish two consecutive packets, the sender adds an additional bit on each packet (called alternating bit because the sender uses 0 and 1 alternatively). 
```

Comment:
I would suggest to write it as  "Alternating Bit Protocol (ABP) is a communication protocol...." instead of "ABP (Alternating Bit Protocol) is a communication protocol...."

Reply:
Solved